nailgun/entities.py

```diff
@@ -3142,6 +3144,44 @@ def summary(self, synchronous=True, timeout=None, **kwargs):
         response = client.get(self.path('summary'), **kwargs)
         return _handle_response(response, self._server_config, synchronous, timeout)
 
+    def cancel(self, synchronous=True, timeout=None, **kwargs):
```

Comment:
Rename to the exact name of endpoint:
```suggestion
    def bulk_cancel(self, synchronous=True, timeout=None, **kwargs):
```